src/actions.js

```diff
@@ -8,10 +8,41 @@ export const initGrid = (width, height) => {
 };
 
 export const TURN_BLUE = "TURN_BLUE";
-export const turnBlue = (x, y) => {
+export const turnBlue = (x, y, clickedNum) => {
```

Comment:
I don't think `clickedNum` is necessary in the actions. Evidence: you're not actually providing the argument in these actions and everything works fine.

![image](https://user-images.githubusercontent.com/257909/29010678-e0d25ade-7ae1-11e7-8d59-82632166e13b.png)

Seems like the reducer is taking care of it.